src/layer/interp.cpp

```diff
@@ -56,26 +62,42 @@ int Interp::forward(const Mat &bottom_blob, Mat &top_blob) const
     if (top_blob.empty())
         return -100;
 
-    if (resize_type == 1)//nearest
+    if (resize_type == 1 || bottom_blob.dims == 1)//nearest
```

Comment:
If bottom_blob is a vector, the resize_type is ignored. move this condition out of resize_type path.

```
if (bottom_blob.dims == 1)
{
    // handle this special case, ignore resize_type

    return 0;
}

// all the following code shall not be changed
if (resize_type == 1)//nearest
{
}

```